docs/hip-7-changes.rst

```diff
@@ -0,0 +1,338 @@
+.. meta::
+  :description: This chapter provides an introduction to the HIP API.
+  :keywords: AMD, ROCm, HIP, CUDA, C++ language extensions
+
+.. _compatibility-changes:
+
+*******************************************************************************
+CUDA compatibility changes to HIP API
+*******************************************************************************
+
```

Comment:
There should be a tittle added here "Behavior changes in HIP Runtime API", which will cover the following four sub-sections:
Update ``hipGetLastError``
Cooperative groups changes
Update ``hipPointerGetAttributes``
Update ``hipFree``

Reply:
Done